frontend/sw360-portlet/src/main/webapp/js/main.js

```diff
@@ -240,9 +240,11 @@ function deleteConfirmed(confirmMessage, confirmCallback) {
         content: confirmMessage,
         confirmButtonClass: 'btn-info',
         cancelButtonClass: 'btn-danger',
-        confirm: function() {
-            confirmCallback();
-        }
+        buttons: {
```

Comment:
I have followed the instructions at https://github.com/craftpip/jquery-confirm/wiki/Migrate-from-v2-to-v3. Also in our old code there was no cancel mentioned in the code. I will check this.

Can you give me a hint where this dialog appears?